app/src/main/java/com/x8bit/bitwarden/data/platform/processor/AuthenticatorBridgeProcessorImpl.kt

```diff
@@ -68,9 +68,14 @@ class AuthenticatorBridgeProcessorImpl(
         }
 
         override fun getSymmetricEncryptionKeyData(): SymmetricEncryptionKeyData? =
-            authenticatorBridgeRepository
-                .authenticatorSyncSymmetricKey
-                ?.toSymmetricEncryptionKeyData()
+            // Only return symmetric encryption info if a user has enabled authenticator sync:
+            if (authenticatorBridgeRepository.isAuthenticatorSyncEnabledForAnyAccounts) {
+                authenticatorBridgeRepository
+                    .authenticatorSyncSymmetricKey
+                    ?.toSymmetricEncryptionKeyData()
```

Comment:
I was picturing that the `authenticatorBridgeRepository. authenticatorSyncSymmetricKey` itself handles the `isAuthenticatorSyncEnabledForAnyAccounts` logic itself. So that this code in this file doesn't actually change.

Reply:
Yeah I debated about that too. Here's what it looks like repository only... seems nice!

64776a8565095cde964bc4791d90a0e2d38c5b08